config/application.rb

```diff
@@ -33,6 +33,10 @@ class Application < Rails::Application
 
     # Don't generate system test files.
     config.generators.system_tests = nil
+    
```

Comment:
```suggestion

```

This (a couple of spaces) is causing a linting violation, hence the failed build

Reply:
Thanks for the spot on that